python/django_bolt/testing/client.py

```
@@ -23,10 +25,65 @@
+    table is locked``. Either way the symptom is an opaque 500 with nothing
+    pointing at the cause.
+
+    Advisory only: a test whose handlers never touch the database works fine
+    inside a transaction, so this must not fail the run.
+    """
+    global _atomic_block_warning_emitted
+    if connections is None or _atomic_block_warning_emitted:
+        return
+    try:
+        open_aliases = [conn.alias for conn in connections.all(initialized_only=True) if conn.in_atomic_block]
+    except Exception as exc:  # unconfigured settings, no databases, etc.
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)